test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -119,7 +119,7 @@ def test_pytorch_inference_torchdata_gpu(
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_inference)
     if test_utils.is_image_incompatible_with_instance_type(pytorch_inference, ec2_instance_type):
         pytest.skip(f"Image {pytorch_inference} is incompatible with instance type {ec2_instance_type}")
-    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.13"):
+    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.13.1"):
```

Comment:
You will need to update this condition for every future patch version of 1.13. Instead, you can just make the upper bound "<1.14"

Reply:
Makes sense as there will be no 1.14 since PT is going to v2.0 next.